File: Clock.cpp
#include <QApplication>
#include <QWidget>
#include <QLabel>
#include <QTimer>
#include <QDateTime>

class DigitalClock : public QWidget {
    Q_OBJECT

public:
    DigitalClock(QWidget *parent = nullptr) : QWidget(parent) {
        timeLabel = new QLabel(this);
        timeLabel->setFont(QFont("Arial", 50));
        timeLabel->setStyleSheet("color: #00FF00; background-color: black;");
        timeLabel->setAlignment(Qt::AlignCenter);

        dayLabel = new QLabel(this);
        dayLabel->setFont(QFont("Ink Free", 25, QFont::Bold));
        dayLabel->setAlignment(Qt::AlignCenter);

        dateLabel = new QLabel(this);
        dateLabel->setFont(QFont("Ink Free", 30));
        dateLabel->setAlignment(Qt::AlignCenter);

        QVBoxLayout *layout = new QVBoxLayout;
        layout->addWidget(timeLabel);
        layout->addWidget(dayLabel);
        layout->addWidget(dateLabel);
        setLayout(layout);

        update();

        QTimer *timer = new QTimer(this);
        connect(timer, &QTimer::timeout, this, &DigitalClock::update);
        timer->start(1000);
    }

private slots:
    void update() {
        QString timeString = QTime::currentTime().toString("h:mm:ss AP");
        timeLabel->setText(timeString);

        QString dayString = QDate::currentDate().toString("dddd");
        dayLabel->setText(dayString);

        QString dateString = QDate::currentDate().toString("MMMM d, yyyy");
        dateLabel->setText(dateString);
    }

private:
    QLabel *timeLabel;
    QLabel *dayLabel;
    QLabel *dateLabel;
};

int main(int argc, char *argv[]) {
    QApplication app(argc, argv);
    DigitalClock clock;
    clock.show();
    return app.exec();
}

#include "main.moc"

File: Clock.java
import javax.swing.*;
import java.awt.*;
import java.text.SimpleDateFormat;
import java.util.Date;

public class TimeDateDisplay {
    public static void main(String[] args) {
        JFrame frame = new JFrame("Time and Date Display");
        frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
        frame.setSize(400, 200);

        JLabel timeLabel = new JLabel("", JLabel.CENTER);
        timeLabel.setFont(new Font("Arial", Font.PLAIN, 50));
        timeLabel.setForeground(Color.GREEN);
        frame.add(timeLabel);

        JLabel dayLabel = new JLabel("", JLabel.CENTER);
        dayLabel.setFont(new Font("Ink Free", Font.BOLD, 25));
        frame.add(dayLabel);

        JLabel dateLabel = new JLabel("", JLabel.CENTER);
        dateLabel.setFont(new Font("Ink Free", Font.PLAIN, 30));
        frame.add(dateLabel);

        frame.setLayout(new GridLayout(3, 1));
        frame.setVisible(true);

        updateTimeDateLabels(timeLabel, dayLabel, dateLabel);
    }

    public static void updateTimeDateLabels(JLabel timeLabel, JLabel dayLabel, JLabel dateLabel) {
        SimpleDateFormat timeFormat = new SimpleDateFormat("hh:mm:ss a");
        SimpleDateFormat dayFormat = new SimpleDateFormat("EEEE");
        SimpleDateFormat dateFormat = new SimpleDateFormat("MMMM dd, yyyy");

        Timer timer = new Timer(1000, e -> {
            String time = timeFormat.format(new Date());
            String day = dayFormat.format(new Date());
            String date = dateFormat.format(new Date());

            timeLabel.setText(time);
            dayLabel.setText(day);
            dateLabel.setText(date);
        });

        timer.setInitialDelay(0);
        timer.start();
    }
}
